kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -112,6 +113,7 @@ private Environment() {
     private static final String CATALOG_SOURCE_NAME_DEFAULT = "kroxylicious-source";
     private static final String KROXYLICIOUS_OLM_DEPLOYMENT_NAME_DEFAULT = "kroxylicious-operator";
     private static final String CATALOG_NAMESPACE_DEFAULT = "openshift-marketplace";
+    public static final String ARCHITECTURE_DEFAULT = "x86_64";
```

Comment:
My default architecture isn't the same as your.    You really mean the default for the test suite.   It is really the kcat based tests with the limitation, so rather than introduce a general 'arch default' thing, let's just make it a kcat test specific thing.